custom_components/powercalc/data/signify/LTA009/model.json

```diff
@@ -0,0 +1,15 @@
+{
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "tuya",
```

Comment:
Could you provide some more information about what exact smart plug you used. This information is good to have for future reference.